lib/travis/build/script/elm.rb

```diff
@@ -3,13 +3,13 @@ module Build
     class Script
       class Elm < NodeJs
         # Default NodeJS version to install
-        DEFAULT_NODE_VERSION = '10.13.0'
+        DEFAULT_NODE_VERSION = '12'
 
         DEFAULTS = {
-          elm: 'elm0.19.0',
+          elm: 'latest-0.19.1',
         }
 
-        ELM_TEST_REQUIRED_NODE_VERSION = '6.0.0'
+        ELM_TEST_REQUIRED_NODE_VERSION = '8.0.0'
```

Comment:
Did `elm-test` version change?

Reply:
elm-test versions match elm language versions. (So elm-test 0.19.1 supports elm tests writing using elm 0.19.1 etc)

Reply:
Right. Unless elm 0.19.1 now requires Node.js 8, there is no reason to change this value.

Reply:
elm-test 0.19.1 does now require node 8:

https://github.com/rtfeldman/node-test-runner/blob/master/package.json#L7

Reply:
Thanks for checking!